libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/Instances.hs

```diff
@@ -879,23 +879,26 @@ instance IsConwayUniv fn => HasSpec fn CoinPerByte
 
 instance HasSimpleRep CostModels
 instance IsConwayUniv fn => HasSpec fn CostModels where
-  emptySpec =
-    Cartesian
-      (constrained $ \m -> size_ (dom_ m) <=. 3)
-      (constrained $ \p -> size_ (dom_ (fst_ $ fromGeneric_ p)) <=. 3)
+  type TypeSpec fn CostModels = ()
+  emptySpec = ()
+  combineSpec _ _ = TrueSpec
+  genFromTypeSpec _ = pureGen arbitrary
+  shrinkWithTypeSpec _ = shrink
+  conformsTo _ _ = True
+  toPreds _ _ = toPred True
+
+-- FIXME: previous implementation no longer builds
+-- emptySpec =
+--   Cartesian
+--     (constrained $ \m -> size_ (dom_ m) <=. 3)
+--     (constrained $ \p -> size_ (dom_ (fst_ $ fromGeneric_ p)) <=. 3)
```

Comment:
@MaximilianAlgehed I am not so sure why my change worked and I have no idea if it is even correct, but all of the tests seem to be happy about the change. I did it by example of other instances.
Could you please check this change whenever you can and potentially fix it if I screwed it up.

Reply:
For now I think it works because we try to avoid generating cost models because they are insanely expensive to generate. So for now this shouldn't be a problem. However, I can have a closer look at fixing this ASAP. For now this shouldn't break anything.

Reply:
Possibly instead of `Arbitrary` you want to use `genValidCostModels` - ther was a comment elsewhere about Arbitrary not working for Conway?

Reply:
>  ther was a comment elsewhere about Arbitrary not working for Conway?

That comment is only relevant for serialization, so using `arbitrary` should be fine here

>  I can have a closer look at fixing this ASAP.

There is no rush on it. Whenever you get a chance